src/main/java/seedu/duke/commands/EventCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.duke.commands;
+
+import seedu.duke.trackables.Event;
+import seedu.duke.trackables.Task;
+
+import java.util.List;
+
+public class EventCommand extends Command {
+
+    private Event event;
+
+    public EventCommand(Event event) {
+        this.event = event;
+    }
+
+    @Override
+    public void execute(List<Task> tasks) {
+        tasks.add(event);
+        echo(new String[] {
```

Comment:
The indentation for wrapped lines in execute method should be 8 spaces.

Reply:
This is the indentation that CheckStyle approves. 

Line 19 at level 8
Line 20-22 at level 12
Line 23 at level 8 (to close the String array instantiation)